e2e-runner/e2e_runner/constants.py

```diff
@@ -25,12 +25,12 @@
     "RouteTables",
 ]
 
-DEFAULT_KUBERNETES_VERSION = "v1.28.4"
+DEFAULT_KUBERNETES_VERSION = "v1.28.10"
```

Comment:
any reason why we should not be having the latest version here as well? v1.03.2?